### DISCUSSION TITLE
Anyway to make the main page into working html site?

### DISCUSSION THREAD

**Discussion by SRoy89** [community/community#22792] [Other Feature Feedback, Questions, & Ideas]
<p>My page is this <a href="https://sroy89.github.io/" rel="noopener nofollow ugc">https://sroy89.github.io</a> which doesn’t show anything, but is there a way to make this a working page?</p>
<p>I basically want to remove <code>/SRoyPortfolio</code> from <a href="https://sroy89.github.io/SRoyPortfolio/" rel="noopener nofollow ugc">https://sroy89.github.io/SRoyPortfolio</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/7bc87c6180a5dc0815bdbf9890c12ca080286305.png" data-download-href="/uploads/short-url/hF2gBB0LVrYzdIWKCaxXFPCZ2pn.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/61577078/181073144-1f183a70-9291-48f2-9964-eafa11cb6b65.png" alt="image" data-base62-sha1="hF2gBB0LVrYzdIWKCaxXFPCZ2pn" width="690" height="399" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/7bc87c6180a5dc0815bdbf9890c12ca080286305_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">746×432 17.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>You’ll need to create a “user site” instead of a project site. See <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites">Types of GitHub Pages sites</a>:</p>
<blockquote>
<p>To publish a user site, you must create a repository owned by your user account that’s named <code>&lt;user&gt;.github.io</code> . […] Unless you’re using a custom domain, user and organization sites are available at <code>http(s)://&lt;username&gt;.github.io</code> or <code>http(s)://&lt;organization&gt;.github.io</code> .</p>
</blockquote>
<p>I don’t know if renaming your repository will work, or if you need to create a new one.</p>

**Comment by otter-computer** [comment]
<aside class="quote no-group" data-username="airtower-luna" data-post="3" data-topic="135630">
<div class="title">
<div class="quote-controls"></div>
 airtower-luna:</div>
<blockquote>
<p>I don’t know if renaming your repository will work, or if you need to create a new one.</p>
</blockquote>
</aside>
<p>Renaming the repository to <code>sroy89.github.io</code> should work, yes!</p>
